runtime/common/src/lib.rs

```diff
@@ -1231,6 +1246,17 @@ macro_rules! impl_config_traits {
             type Currency = Balances;
             type WeightInfo = zrml_styx::weights::WeightInfo<Runtime>;
         }
+
+        impl zrml_neo_swaps::Config for Runtime {
+            type CompleteSetOperations = PredictionMarkets;
+            type ExternalFees = MarketCreatorFee<Runtime>;
+            type MarketCommons = MarketCommons;
+            type MultiCurrency = AssetManager;
```

Comment:
That's because `MultiCurrency` doesn't implement `ZeitgeistAssetManager`.

Reply:
I don't understand this. MultiCurrency here is equal to AssetManager and the trait ZeitgeistAssetManager isn't really useful for us, right? It has been before for the destruction of a whole asset class, but since we know the iteration over all accounts is too expensive, we don't use it anymore extensively, or do we?

Reply:
But anyways, if you think the name fits better, because it follows the convention of using the pallets name, I am fine with that!

Reply:
> It has been before for the destruction of a whole asset class, but since we know the iteration over all accounts is too expensive, we don't use it anymore extensively, or do we?

Interesting point. Yeah, maybe we can get rid of the entire trait now.